### DISCUSSION TITLE
[Bug report] Large author pages extend beyond the upper screen edge in Simple Theme 2.8.3.0

### DISCUSSION THREAD

**Discussion by Yora0** [GetPublii/Publii#1424] [Bug Report]
### Operating system

Fedora 39

### Publii version

0.45.1

### Issue type

Free theme

### Bug description

On my site using the default Simple theme, the author page text extends above the upper window edge. The text is all there, but it is rendered outside of the window area in both Pale Moon 33.0.1, Firefox 123.0.1, Opera 108.0.5067.20, and Falcon 23.08.5.

So probably a css issue and not a browser issue.

I just installed Publii and created the site. I have not touched any custom css or html yet, and I have no other themes or plugins installed. So I assume this would be a universal issue for any sites that use the Simple theme and have an author page longer than a few lines. Can anyone confirm? (And of course, share a fix if one is known.)
![Screenshot_20240310_153120](https://github.com/GetPublii/Publii/assets/162896858/f0866a4e-c926-4cbd-ae1e-890a0bda9518)

### Steps to reproduce

Create an author page that consists of several paragraphs.
Set the page theme to Simple (v.2.8.0.3).
Preview the site and open the author page.

### Relevant log output

In inspecting the page in the browser, <div class=hero> has a height of 173.2.
<div class=hero__content> has a height of 1051.7.
Obviously this causes content to be clipped outside of frame.

**Comment by bobmitro** [comment]
Use a shorter content; the author description field is meant for brief descriptions. Instead, create a dedicated post for the author's bio.

**Comment by candidexmedia** [comment]
If you're comfortable doing so: can you share a live link to the page?

If you're keen on keeping the longer text, it may just be a matter of adding some CSS tweaks, but that's easier to solve with a live page.
